docs/environment.md

```diff
@@ -1,5 +1,9 @@
 # Setting up your environment
 
+## Prerequisites
+
+If you are planning to contribute to the solution, your OS environment will need [git](https://git-scm.com/) and [git lfs](https://git-lfs.github.com/) installed. Depending on the OS that you are running the installation instructions may vary. Please refer to respective documentation sections on the tools' websites for detailed instructions. 
```

Comment:
You need git and git-lfs also if you just want to check the code out.

Reply:
Updated